condor/migration-guide.md

```diff
@@ -0,0 +1,193 @@
+
+While it may not be strictly necessary to upgrade your integration if all you do is submit Deploys to the network (e.g. transfers in and out of accounts), it is STRONGLY RECOMMENDED to upgrade to the new SDK now, to experience a seamless upgrade to Casper 2.0 and beyond.
+
+>[!IMPORTANT]
+> **Addressable Entity is not enabled in Casper 2.0.**
+> 
+> Updating to a Casper 2.0-compatible SDK will be necessary when the **addressable entity** is enabled.
+> 
+> Once the **addressable entity** is enabled, exchanges will be required to use an SDK compatible with the Entity.
+
+
+Please see instructions below for your specific SDK language:
+
+| SDK    | Instructions | 
+|---------------------|-------------|
+| **JavaScript** | The appropriate SDK to upgrade to is the latest v5.x release can be found [here](https://github.com/casper-ecosystem/casper-js-sdk/releases). |
+| | Common use case examples (connect to RPC, send a transfer, etc.) can be found in [Casper JS SDK 5.0 README](https://github.com/casper-ecosystem/casper-js-sdk/tree/feat-5.0.0?tab=readme-ov-file#usage-examples) |
+| |To upgrade your integration, follow the detailed [V2 to V5 Migration Guide](https://github.com/casper-ecosystem/casper-js-sdk/blob/feat-5.0.0/resources/migration-guide-v2-v5.md) | 
+| **#C .NET** | The appropriate SDK to upgrade to is the latest v3.x release can be found [here](https://github.com/make-software/casper-net-sdk/releases).|
+| | Typical use case examples (connect to RPC, send a transfer, etc.) can be found in [the Examples folder](https://github.com/make-software/casper-net-sdk/tree/casper-2.0.0/Docs/Examples). |
+| |To upgrade your integration, follow the document [Migration from Casper .NET SDK v2.x to v3.0](https://github.com/make-software/casper-net-sdk/blob/casper-2.0.0/Docs/Articles/Casper20MigrationGuide.md) | 
+| **Go** | The appropriate SDK to upgrade to is the latest v2.x release found [here](https://github.com/make-software/casper-go-sdk/releases).|
+| |Typical use case examples (connect to RPC, send a transfer, etc.) can be found in the [README](https://github.com/make-software/casper-go-sdk/blob/master/README.md). | 
+| **Java** | The appropriate SDK to upgrade to is the latest v2.x release can be found [here](https://github.com/casper-network/casper-java-sdk/releases).|
+| |Casper Java SDK in Maven Repository can be found in the link [here](https://central.sonatype.com/artifact/network.casper/casper-java-sdk/2.7.0-BETA.4/overview). | 
+| **Other** | If you are using an SDK not listed in this chapter, please contact the Casper Exchange Support Team.|
```

Comment:
Is there any reason why we use a table here? It's not formatted conventionally. Having subsections for each language would probably be more readable:

<img width="983" alt="Screenshot 2025-03-06 at 13 53 08" src="https://github.com/user-attachments/assets/b0746b69-2085-4a94-b8c9-d990307cfe9a" />

Additionally, the last row says to contact the Casper team but doesn't provide any way how to do that.